Tests/RxCocoaTests/DelegateProxyTest.swift

```diff
@@ -770,3 +774,13 @@ extension MockTestDelegateProtocol
 {
 }
 #endif
+
+#if os(iOS) || os(macOS)
+import WebKit
+
+@available(iOS 10.0, OSXApplicationExtension 10.10, *)
+extension MockTestDelegateProtocol
+    : WKNavigationDelegate
+{
+}
+#endif
```

Comment:
Same, probably not needed

Reply:
same as https://github.com/ReactiveX/RxSwift/pull/2144#discussion_r391470310

Reply:
Still somewhat confused - if you have tests in a separate file, maybe this conformance should be there, and not in the general Delegate Proxy Test file.

Reply:
So can we move this to **DelegateProxyTest+WebKit.swift** ?

Reply:
sure makes sense.

I felt safer developing the delegate proxy for the first time having a test confirming I did it right, you know what I mean?

Reply:
Yup, that's fair. :)